### PR TITLE
Add new top menu to the app +

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
+ transform landing page from **modal** to a **regular page**

image

### CODE REVIEW

File: src/app/components/landing-page/landing-page-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  public showLoginButton: boolean;
+  public showLogOutButton: boolean;
+  public authentMode = 'false';
+
+  public isAuthenticated = false;
+  public isAuthentActivated = false;
+  public name: string;
+  public avatar: string;
+  public orgs: UserOrgData[] = [];
+  public currentOrga = '';
```
Comment (QuCMGisaia) [inline src/app/components/landing-page/landing-page-dialog.component.ts:73]: unused ?

File: src/app/components/landing-page/landing-page.component.ts [hunk 2]
```diff
@@ -16,127 +16,100 @@ KIND, either express or implied.  See the License for the
...
-  load = 2
-}
 
 export interface Configuration {
   name: string;
   last_update_date: any;
   actions: Array<ConfigAction>;
 }
 
+enum InitialChoice {
```
Comment (QuCMGisaia) [inline src/app/components/landing-page/landing-page.component.ts:51]: Also defined in the dialog

File: src/app/components/landing-page/landing-page-dialog.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/components/landing-page/landing-page-dialog.component.ts:73]: true

File: src/app/app.component.scss [hunk 3]
```diff
@@ -16,11 +16,43 @@ KIND, either express or implied.  See the License for the
...
+}
+
+::ng-deep .top-menu {
+  border-bottom: 1px solid #dadada;
 }
+
+.top-left-menu {
+  display: flex;
+  align-items: center;
+  justify-content: spa;
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply src/app/app.component.scss:38]: bien vu

**Pull request merged**